docs/source/getting-started/content/assembly_troubleshooting.adoc

```diff
@@ -1,4 +1,10 @@
 [id="troubleshooting-codeready-containers_{context}"]
 = Troubleshooting {rh-prod}
 
+[NOTE]
+====
+{rh-prod}'s goal is to deliver an OpenShift environment for development and testing purposes.
+Issues occurring during installation or usage of specific OpenShift applications are outside of {prod}'s realm and should be reported to the relevant project.
```

Comment:
we might need to be more specific and point to the actual locations to report issues for OpenShift itself, etc.

Reply:
Can add an OpenShift pointer, but OpenShift applications will be spread among a number of locations, so hard to give any pointers for that.

Reply:
Yes, there are lot of openshift projects/components, it is really hard to put any specific pointer.